### ISSUE TITLE
Fix crash on mod auto

### ISSUE THREAD

**Issue by GSculerlor** [GSculerlor/Cytosu#7]
Leftover from initial implementation (b11b58f47b16e2ffbb3d1fdd3131ad8cf52750a3). Might add a test as well

**Comment by WEDeach** [comment]
https://www.youtube.com/watch?v=8K3FsO8RQ4Q
Just let you know in auto mode, it may not be so perfect
Each playback has a different result, even if I didn’t use rewind

**Comment by GSculerlor** [comment]
Yes, the current implementation is basically my "hacky" way to achieve auto mod. The next update will fix this issue! Kinda busy with other stuff so maybe will come later this week or next week.

**Comment by NekomusuMae** [comment]
I had trouble whether it should be said on this issue or a new one but can we swap the positions of Relax and Autoplay to stay consistent with every other ruleset (custom or not).
Cytosu:
![image](https://user-images.githubusercontent.com/43427333/119236005-bb2fe280-bafa-11eb-976d-61b286310f9b.png)
Every other ruleset (this case osu!standard):
![image](https://user-images.githubusercontent.com/43427333/119236021-c97dfe80-bafa-11eb-9615-dd4c5e645e02.png)

**Comment by GSculerlor** [comment]
Oopsie! didn't notice that until now! Fixed on latest release (https://github.com/GSculerlor/Cytosu/releases/tag/2021.523.0)
![osu_2021-05-23_00-55-36](https://user-images.githubusercontent.com/6506864/119236675-f49c3880-bb62-11eb-9356-07ba9e685d4f.png)
